sentry-android/src/main/java/io/sentry/android/AndroidSentryClientFactory.java

```diff
@@ -71,28 +69,6 @@ public SentryClient createSentryClient(Dsn dsn) {
     }
 
     @Override
-    public Dsn lookupDsn() {
-        String stringDsn = "";
-
-        // attempt to get DSN from AndroidManifest
-        ApplicationInfo appInfo = null;
-        try {
-            PackageManager packageManager = ctx.getPackageManager();
-            appInfo = packageManager.getApplicationInfo(ctx.getPackageName(), PackageManager.GET_META_DATA);
-            stringDsn = appInfo.metaData.getString("io.sentry.android.DSN");
-        } catch (PackageManager.NameNotFoundException e) {
-            // skip
-        }
-
-        if (TextUtils.isEmpty(stringDsn)) {
-            throw new NullPointerException("Sentry DSN is not set, you must provide it to"
-                + "Sentry.init or in your Android manifest file.");
-        }
-
-        return new Dsn(stringDsn);
-    }
```

Comment:
Dropping the manifest thing in favor of,

1. Setting it in your code `Sentry.init(dsn)`
2. The coming `sentry.properties` config file.